.xo-config.js

```diff
@@ -44,7 +44,28 @@ module.exports = {
 		// Temporarily disabled
 		// TODO: Need to decide what to do with this rule. Either keep it disabled here or remove it from XO.
 		'@typescript-eslint/no-invalid-void-type': 'off', // Reason: https://github.com/sindresorhus/refined-github/pull/3168#discussion_r434479434
-		'unicorn/no-fn-reference-in-iterator': 'off'
+		'unicorn/no-fn-reference-in-iterator': 'off',
+		'@typescript-eslint/no-extra-non-null-assertion': 'error',
+		'@typescript-eslint/consistent-type-definitions':'error',
+		'@typescript-eslint/require-await': 'error',
+		'@typescript-eslint/explicit-function-return-type': [
+			'error',
+			{
+				allowExpressions: true,
+				allowTypedFunctionExpressions: true,
+				allowHigherOrderFunctions: true,
+				allowConciseArrowFunctionExpressionsStartingWithVoid: false
+			}
+		],
+		'@typescript-eslint/explicit-module-boundary-types': [
```

Comment:
Just make sure you're ok with this: https://github.com/xojs/eslint-config-xo-typescript/blob/7842d05deaadff1ced9a46f3e56d786e7b2922d2/index.js#L136-L138

Reply:
According to [its docs](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-module-boundary-types.md#when-not-to-use-it), this rule is only a "less strict" version of `explicit-function-return-type`, which we already use. Right?

If so, it should be dropped from here and from XO’s comments